packages/shared/lib/services/proxy.service.ts

```diff
@@ -170,6 +170,12 @@ class ProxyService {
             Object.keys(data as any).forEach((key) => {
                 formData.append(key, (data as any)[key]);
             });
+            for (const file of externalConfig.files || []) {
```

Comment:
I think you can remove the above 

```
            Object.keys(data as any).forEach((key) => {
                formData.append(key, (data as any)[key]);
            });
```
as this was my quick attempt to add in the support for file uploads in https://github.com/NangoHQ/nango/pull/1584

Reply:
multipart/form-data data can also contain key/value pairs (ie: form input values) that are not files but end up in the req.body and therefore data, so we need to do both.

Reply:
Cool